consensus/src/state_computer.rs

```diff
@@ -473,3 +481,149 @@ async fn test_commit_sync_race() {
     assert!(executor.sync_to(generate_li(2, 8)).await.is_ok());
     assert_eq!(*recorded_commit.time.lock(), LogicalTime::new(2, 8));
 }
+
+const PIPELINE_DEPTH: usize = 8;
+
+struct ExecutionPipeline {
```

Comment:
May be better to move it to its own module?

Reply:
done